src/frontend/src/flows/manage/authenticatorsSection.ts

```diff
@@ -95,3 +96,45 @@ export const devicesSection = ({
       </div>
     </aside>`;
 };
+
+export const authenticatorItem = ({
+  authenticator,
+  index,
+}: {
+  authenticator: DedupAuthenticator;
+  index: number;
+}) => {
+  const remove = authenticator.remove;
+
+  return html`
+    <li class="c-action-list__item" data-device=${authenticator.alias}>
+      <div class="c-action-list__label">
+        ${authenticator.alias}
+        ${authenticator.dupCount !== undefined && authenticator.dupCount > 0
```

Comment:
`> 0` is de facto not undefined or is typescript complaining?

Reply:
ah good question, I assumed it would complain but maybe not. if it doesn't complain I'll change.

Reply:
>         TS18048: 'dupCount' is possibly 'undefined'.